### ISSUE TITLE
'Cannot fetch from Scholar' Error

### ISSUE THREAD

**Issue by tdrivas** [scholarly-python-package/scholarly#472]
Dear all, 

I was using scholary to fetch automatically my papers' citations every 2-3 days. Before some weeks, the same code yields the error "Cannot fetch from Google". I tested the same code in another VM machine outside my country (thus different IP) and it gives the same error. Afterwards, I added to the code the following lines: 

```
pg = ProxyGenerator()
pg.FreeProxies()
scholarly.use_proxy(pg)  
```

but no luck again. I need to highlight that i am using the function search_author_id to get the data. 

Any idea?

Thanks in advance! 
Best,
Thanassis

**Comment by Kalaweksh** [comment]
Also experiencing this issue. The exact error is "scholarly._proxy_generator.MaxTriesExceededException: Cannot Fetch from Google Scholar."

**Comment by arunkannawadi** [comment]
Marking this as a duplicate of #465 where all pages are being blocked by Google Scholar unless a robust proxy is used.
